app/component/map/tile-layer/SelectCityBikeRow.js

```diff
@@ -26,7 +26,7 @@ function SelectCityBikeRow({ selectRow, name, networks, id }, { config }) {
             <Icon img={img} />
           </div>
           <span className="choose-row-center-column">
-            <h5 className="choose-row-header">{name}</h5>
+            <h5 className="choose-row-header">{name ?? `Station - ${id}`}</h5>
```

Comment:
Maybe better use ``name || `Station - ${id}`` if name is empty

Reply:
You are absolutely right. I'm changing it right now.